.gitignore

```diff
@@ -0,0 +1,2 @@
+
+__pycache__/config.cpython-36.pyc
```

Comment:
change it to `__pychache__`, because right now it is only ignoring one file, not the whole folder.

Reply:
done